public/static/docs/api-reference/get_url.md

```diff
@@ -0,0 +1,36 @@
+# dvc.api.get_url()
+
+Returns the full URL to the <abbr>data artifact</abbr> specified by its `path`
+in a `repo` (<abbr>DVC project</abbr>).
+
+## Signature
+
+```py
+get_url(path, repo=None, rev=None, remote=None)
+```
+
+## Parameters
+
+- **`path`** - used to specify the location of the target artifact within the
+  source project in `repo`, relative to the project's root.
+
+- `repo` - specifies the location of the source DVC project. Both HTTP and SSH
+  protocols are supported for online Git repositories (e.g.
+  `[user@]server:project.git`). `repo` can also be a local file system path to
+  an "offline" project.
```

Comment:
`repo` can be a path to a local DVC project right? Even if it's not actually a Git repo? (i.e. created with `dvc init --no-scm`)

Reply:
And when omitted, its effective value is `.`, I assume?

Reply:
> repo can be a path to a local DVC project... And when omitted, its effective value is . ?

Yes and yes.

Reply:
But if there's no remote configured, a `NoRemoteError` is thrown. Commented in 8b133caf.